### PR TITLE
Showing the CSS for the synth keyboard example

### PR DESCRIPTION
**Pull request by Anurella**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'
 #7466

> What was wrong/why is this fix needed? (quick summary only)
The styling for the keyboard was not visible on page.

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/api/web_audio_api/simple_synth/index.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h5 id="The_waveform_picker">The waveform picker</h5>
   &lt;/div&gt;
 &lt;/div&gt;</pre>
 
-<div class="hidden">
+<div class="">
```
Comment (hamishwillee) [inline files/en-us/web/api/web_audio_api/simple_synth/index.html:76]: If you remove the last class in a div then consider whether the whole div is needed - ie does it do layout any more. If not,  as in this case, you should remove the `<div>` and `</div>` tags as well.

File: files/en-us/web/api/web_audio_api/simple_synth/index.html [hunk 2]
```diff
@@ -450,7 +453,7 @@ <h3 id="Building_the_keyboard">Building the keyboard</h3>
  <li>For each key in the octave, we check to see if the note's name has more than one character. We skip these, because we're leaving out the sharp notes in this example. If the note's name is only one character, then we call <code>createKey()</code>, specifying the note string, octave, and frequency. The returned element is appended to the octave element created  in step 4.</li>
  <li>When each octave element has been built, it's appended to the keyboard.</li>
  <li>Once the keyboard has been constructed, we scroll the note "B" in octave 5 into view; this has the effect of ensuring that middle-C is visible along with its surrounding keys.</li>
- <li>Then a new custom waveform is built using {{domxref("AudioContext.createPeriodicWave()")}}. This waveform will be used any time the user selects "Custom" from the waveform picker control.</li>
+ <li>Then a new custom waveform is built using {{domxref("BaseAudioContext.createPeriodicWave")}}. This waveform will be used any time the user selects "Custom" from the waveform picker control.</li>
```
Comment (hamishwillee) [inline files/en-us/web/api/web_audio_api/simple_synth/index.html:456]: Any particular reason to remove the `()`? They are used to indicate that this is a function.

Comment (Anurella) [reply files/en-us/web/api/web_audio_api/simple_synth/index.html:456]: I did not make any amends to this section, but you can highlight what needs to be corrected

File: files/en-us/web/api/web_audio_api/simple_synth/index.html (hunk 1 above)
Comment (Anurella) [reply files/en-us/web/api/web_audio_api/simple_synth/index.html:76]: Noted, thank you.

File: files/en-us/web/api/web_audio_api/simple_synth/index.html (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/web/api/web_audio_api/simple_synth/index.html:456]: The text `{{domxref("BaseAudioContext.createPeriodicWave")}}` should change to {{domxref("BaseAudioContext.createPeriodicWave()")}}

File: files/en-us/web/api/web_audio_api/simple_synth/index.html (hunk 1 above)
Comment (jpmedley) [reply files/en-us/web/api/web_audio_api/simple_synth/index.html:76]:
```suggestion
<div>
```

File: files/en-us/web/api/web_audio_api/simple_synth/index.html (hunk 2 above)
Comment (jpmedley) [reply files/en-us/web/api/web_audio_api/simple_synth/index.html:456]:
```suggestion
 <li>Then a new custom waveform is built using {{domxref("BaseAudioContext.createPeriodicWave()")}}. This waveform will be used any time the user selects "Custom" from the waveform picker control.</li>
```

File: files/en-us/web/api/web_audio_api/simple_synth/index.html [hunk 3]
```diff
@@ -506,7 +509,7 @@ <h4 id="Playing_a_tone">Playing a tone</h4>
 }
 </pre>
 
-<p><code>playTone()</code> begins by creating a new {{domxref("OscillatorNode")}} by calling the {{domxref("AudioContext.createOscillator()")}} method. We then connect it to the main gain node by calling the new oscillator's {{domxref("OscillatorNode.connect()")}} method;, which tells the oscillator where to send its output to. By doing this, changing the gain of the main gain node will affect the volume of all tones being generated.</p>
+<p><code>playTone()</code> begins by creating a new {{domxref("OscillatorNode")}} by calling the {{domxref("BaseAudioContext.createOscillator")}} method. We then connect it to the main gain node by calling the new oscillator's {{domxref("OscillatorNode.connect()")}} method;, which tells the oscillator where to send its output to. By doing this, changing the gain of the main gain node will affect the volume of all tones being generated.</p>
```
Comment (jpmedley) [inline files/en-us/web/api/web_audio_api/simple_synth/index.html:512]:
```suggestion
<p><code>playTone()</code> begins by creating a new {{domxref("OscillatorNode")}} by calling the {{domxref("BaseAudioContext.createOscillator()")}} method. We then connect it to the main gain node by calling the new oscillator's {{domxref("OscillatorNode.connect()")}} method;, which tells the oscillator where to send its output to. By doing this, changing the gain of the main gain node will affect the volume of all tones being generated.</p>
```

**Comment by Anurella** [comment]
@jpmedley @hamishwillee I have applied the suggested review to the file.

File: files/en-us/web/api/web_audio_api/simple_synth/index.html (hunk 1 above)
Comment (Anurella) [reply files/en-us/web/api/web_audio_api/simple_synth/index.html]: removed the div wrapper

**Comment by wbamberg** [comment]
@hamishwillee , @jpmedley , I've fixed the merge conflict and it looks like the changes you've asked for have been made. Could you re-review and merge if it looks ready?

**Review by hamishwillee — APPROVED** [review]
Yes, fixed. Thanks!

**Pull request merged**